core-web/libs/sdk/angular/src/lib/dotcms-client-token.ts

```diff
@@ -0,0 +1,23 @@
+import { EnvironmentProviders, InjectionToken, makeEnvironmentProviders } from '@angular/core';
```

Comment:
Does this file needs to be in a folder?

Reply:
This was removed, now is inside angular example